common/config/azure-pipelines/jobs/version-bump.yaml

```diff
@@ -154,149 +240,81 @@ jobs:
-                # it is likely that any other thing that was being pulled from master probably also needs to be pulled from release branch instead
-                sed -i "s/$search/$replace/g" "$docsYamlPath"
-                git add "$docsYamlPath"
-                git commit -m "Update 'gather-docs.yaml' to previous release branch name"
-              fi
-            displayName: Update 'gather-docs.yaml' and commit to release branch
-
-          - bash: git push --set-upstream https://$(GITHUBTOKEN)@github.com/iTwin/itwinjs-core $(releaseBranchName) -q
-            displayName: Publish the release branch
-
-  - ${{ if eq(parameters.BumpType, 'releaseCandidate') }}:
-      - job: UpdateMaster
-        displayName: Update master to next minor version
-        dependsOn: CreateBranch
-        condition: and(eq(dependencies.CreateBranch.result, 'Succeeded'), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
-
-        steps:
-          - checkout: self
-          - task: NodeTool@0
-            displayName: "Use Node 20.x"
-            inputs:
-              versionSpec: 20.x
-              checkLatest: true
-
-          - bash: |
-              git config --local user.email 38288322+imodeljs-admin@users.noreply.github.com
-              git config --local user.name imodeljs-admin
-            displayName: "Setup Git"
-
-          - bash: git checkout --track origin/$(Build.SourceBranchName)
-            displayName: Switching to source branch
-
-          # Running this rush command will bump the version number to the next minor version, which in turn deletes everything within common/change.
-          - bash: node common/scripts/install-run-rush version --override-bump minor --version-policy prerelease-monorepo-lockStep --bump
-            displayName: Update to new dev version on master
-
-          # Only add the changes which delete the changelogs, we do not want them on master anymore.
-          - bash: git add common
-            displayName: "Add changelog deletion"
-
-          - bash: |
-              # Resets all changes, other than the deletion of the changelogs.
-              git checkout -- .
-
-              # Cleans up all untracked files. This could happen if there are new packages that do not have change logs yet.
-              git clean -f -d
-
-              git status
-            displayName: Reset all changes besides changelog
-
-          # The real rush command that we would like to run.
-          # Sets the version number to the next version's first pre-release version.
-          - bash: node common/scripts/install-run-rush version --override-bump preminor --version-policy prerelease-monorepo-lockStep --bump --override-prerelease-id dev
-            displayName: Rush version to new pre-release version
+        displayName: Push version bump for non-release candidate
+        condition: ne('${{ parameters.BumpType }}', 'releaseCandidate')
```

Comment:
So this means that the current workflow of version bump with 'releaseCandidate' for the first RC, then version bump on the release pipeline with 'nightly' still works, correct? @DanRod1999

Reply:
should still work